apps/ui/src/views/Landing.vue

```diff
@@ -20,10 +20,26 @@ const SOCIALS = [
     icon: ICGithub
   }
 ];
+
+const router = useRouter();
+const { web3, authInitiated } = useWeb3();
+
+watch(
+  () => web3.value.account,
+  account => {
+    if (!account) return;
+
+    router.push({ name: 'my-home' });
+  },
+  { immediate: true }
+);
 </script>
 
 <template>
-  <div>
+  <div v-if="!authInitiated || web3.authLoading" class="p-4">
```

Comment:
If metamask is locked

<img width="848" alt="Untitled 4" src="https://github.com/user-attachments/assets/3b75ad1b-60c7-432d-98e4-2562995c414a">

Is it expected? feels bit weird

Reply:
why to show a loader here 🤔

Reply:
To not flash the landing page. 

Else, it will show the landing page, while we're connecting the user async

Reply:
We should have some localstorage variable (add value when user connects and delete if the user logout) we should always redirect to `/home` if user have this. else show `/`

We should handle user wallet connection on `/home`, if user fails to connect his wallet, we should delete localstorage variable and redirect back to `/`

(also we should not try to auto connect if user doesn't have this localstorage value, this may solve https://github.com/snapshot-labs/sx-monorepo/issues/478)

Reply:
I think the issue is that we can not differentiate if the user is ready to be logged or if Metamask / user wallet is locked, this cause an issue where Metamask or WalletConnect will be triggered for login on first page load. Wagmi may fix that, or it could be with a change on Lock.

Reply:
The ideal behavior would be similar than Uniswap, where user dont get Metamask or their wallet triggered if it's locked, but still autoconnect if it's not locked

Reply:
This ideal behavior depends on some changes on lock side, and will be on another PR